docs/introduction.rst

```diff
@@ -101,6 +101,12 @@ though extending Tendermint to handle non-determinism remains for future work.
 with additional features for a name-registry, permissions, and native contracts, and an alternative blockchain API.
 It uses Tendermint as its consensus engine, and provides a particular application state.
 
+`Hydrachain <https://github.com/HydraChain/hydrachain>`__ is a Pyethereum adapted to a non-BFT consensus algorithm modelled on Tendermint.
```

Comment:
I thought we were removing these

Reply:
the reference removed in https://github.com/tendermint/tendermint/pull/661/commits/6a8945f33791626185516deadf649e4ff2103c23 ... this stuff is in the "Tendermint vs. X section of the intro. So maybe we need to do a better job of comparing them, rather than simply mentioning.